sdk/identity/identity-broker/README.md

```diff
@@ -101,7 +101,40 @@ main().catch((error) => {
   process.exit(1);
 });
 ```
-For an example of using an Electron app for retrieving a window handle, see [this sample](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity-broker/samples/v1/typescript/src/index.ts).
+
+For a complete example of using an Electron app for retrieving a window handle, see [this sample](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity-broker/samples/v1/typescript/src/index.ts).
+
+### Use the default account for sign-in
+
+When the `useDefaultBrokerAccount` option is set to `true`, the credential will attempt to silently use the default broker account. If using the default account fails, the credential will fall back to interactive authentication.
+
+```typescript
+import { nativeBrokerPlugin } from "@azure/identity-broker";
+import { useIdentityPlugin, InteractiveBrowserCredential } from "@azure/identity";
+
+useIdentityPlugin(nativeBrokerPlugin);
+
+async function main() {
+  const credential = new InteractiveBrowserCredential({
```

Comment:
do you think we should add client id and tenant id too in the sample to prevent it using the default client id?